apps/app/src/views/routes/Settings/components/templates/TemplateActions.tsx

```diff
@@ -1,24 +1,31 @@
 import {
-  IconButton,
   HStack,
+  IconButton,
   Menu,
   MenuButton,
   MenuItem,
   MenuList,
-  useColorMode,
-  useBreakpointValue
+  useColorMode
 } from '@chakra-ui/react';
 import { FiEdit, FiEye, FiMoreVertical, FiTrash } from 'react-icons/fi';
 
 import { usePhoton } from '@photonhealth/react';
 
+import { PrescriptionTemplate } from 'packages/sdk/dist/types';
```

Comment:
thoughts in general on `import type { ...yadayada...`

Reply:
oh yeah, this is actually a bad import. We should export all types from the sdk at the top level. `Nx` doesnt like when you reach across module boundaries

Reply:
I'll fix